tests/helper/helper_documentation.go

```diff
@@ -0,0 +1,80 @@
+package helper
+
+import (
+	"bufio"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"unicode"
+
+	. "github.com/onsi/gomega"
+)
+
+const (
+	timePatternInOdo = `(\[[0-9smh]+\])` // e.g. [4s], [1m], [3ms]
+	// Credit: https://github.com/acarl005/stripansi/blob/master/stripansi.go
+	ansiPattern          = "[\u001B\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))"
+	unicodeSpinnerFrames = "◓◐◑◒"
+)
+
+// ReplaceTimeInString replaces the time taken to download a Devfile or a starter project for an odo command with a custom value;
+// this function is helpful because the time value is variable and replacing it with the value in mdx content helps in comparing.
+func ReplaceTimeInString(docString string, timeString string) string {
```

Comment:
Please rename to `ReplaceTimesInString` (plural), as you are using `ReplaceAllString`